docs/PR_DESCRIPTION.md

```
@@ -0,0 +1,138 @@
+| Area | Tests | Status |
+|------|-------|--------|
+| Constructor | 4 | ✅ All passing |
+| getModel | 3 | ✅ All passing |
+| convertMessagesForMiMo | 10 | ✅ All passing |
+| createMessage | 19 | ✅ All passing |
+| completePrompt | 9 | ✅ All passing |
+| **Total** | **45** | ✅ **All passing** |
+
+## Test Results
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)